README.md

```diff
@@ -1 +1,360 @@
 # Centrapay Documentation
+
+## Overview
+
+### Introduction
+
+Welcome to Centrapay! Payments through Centrapay are accomplished via a two-step process, involving both the merchant and consumer.  
+
+The API is designed to support payment terminals, point of sale, shopping cart, vending and unattended transactions.
+
+It enables consumers to pay with a range of digital assets, including bitcoin and vouchers. The merchant can initiate payment from a consumer or the consumer can push a payment to the merchant. It will also be possible to activate our services in the payment flow such as carbon offsetting.
```

Comment:
This part can probably be removed and mention the digital assets we support, and the payment flows we support elsewhere and with a simple flow chart for both situations.